cli/flox/doc/include/environment-options.md

```diff
@@ -1,6 +1,10 @@
 ## Environment Options
 
- The following option is supported by environment commands:
+If no environment is specified for an environment command, the environment in
+the current directory or the last activated environment is used.
 
-(-e|--environment) `<name>`
-:    Selects flox environment to be modified or used.  If not provided then flox will fall back to using the default environment.
+`-d`, `--dir`
+:   Path containing a .flox/ directory.
```

Comment:
```suggestion
:   Path to a `.flox` directory.
```
Since `-d` can be used for creating environments

Reply:
I don't think `environment-options.md` should be included by any commands that create an environment. `init` and `pull` have their own descriptions for `-d`

(I just realized I'd included environment-options in pull, but dropping that)